templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -651,29 +651,47 @@ contents:
       echo "${physical_interface}"
     }
 
-    # Accepts parameters $iface, $iface_default_hint_file, $ip_hint_file
-    # Finds the nodeip interface from the interface that matches the ip address in $ip_hint_file.
-    # Otherwise fallbacks to a previously used interface or to the default interface.ç
-    # Never use the interface that is provided inside extra_bridge_file for br-ex1.
-    # Never use br-ex1.
-    # Read $ip_hint_file and return the interface that matches this ip.  Otherwise:
-    # If the default interface is br-ex, use that and return.
+    # Accepts parameters $iface_default_hint_file, $extra_bridge_file, $ip_hint_file, $default_bridge_file
+    # Determines the interface to be used for br-ex. Order of priority is:
+    # 1. Use the user specified interface if provided in the default bridge file
+    # 2. Use the node IP hint interface
+    # 3. Use the previously selected interface
+    # 4. Use the interface detected as default gateway
+    #
+    # Read $default_bridge_file and return contained the interface. Otherwise,
```

Comment:
Sorry, I had a typo in the text I proposed to you
`return the contained interface`